_posts/blog/2016-07-18-subjects.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: "Subjects - neither good nor evil"
+modified:
+categories: blog
+author: tomek_polanski
+excerpt: Get to know the thread switching operations
+tags: [RxJava]
+image:
+date: 2016-07-18T13:00:55-01:00
+---
+
+
+# Subjects - neither good nor evil
```

Comment:
I would capitalise the letters in the titles
http://www.skillsyouneed.com/write/capital-letters.html